### ISSUE TITLE
onReadyScript for multiple selectors

### ISSUE THREAD

**Issue by lancetharper** [garris/BackstopJS#511]
I'm trying to capture a set of buttons in both normal and hover states.

I have a scenario with an array of selectors that works great for the normal state.

I have another scenario with the same array of selectors and an onReadyScript that moves the mouse to the selector to see the hover state. 

The onReadyScript only runs once (obviously) per scenario.

Is there a way to run the hover script for each selector in the scenario or do you have to make one scenario per selector in order to achieve this?

**Comment by garris** [comment]
You will need 1 scenario per -- if you do a lot of this kind of testing you might want to create a js-based config file to generate this for you.  If it's just 10 or less buttons then just specify the scenarios.

**Comment by lancetharper** [comment]
Do you think this would be a difficult feature to add? I can probably help out if you think it's reasonable.

**Comment by garris** [comment]
Yes.  I think this is totally doable.   There is already a clickAndHoverHelper here...
https://github.com/garris/BackstopJS/blob/master/capture/engine_scripts/clickAndHoverHelper.js

I have been meaning to create an example for it.  

You might be able to create something which works similarly but does it at the config or beforeScript level and works similarly to the selctorExpansion feature.

Please ping me with some ideas if you have any.
